packages/apps/desktop-viewer-test/README.md

```diff
@@ -0,0 +1,28 @@
+# Desktop Viewer Sample
+
+This app is an example of using the iTwin Viewer in a modularized React + Electron desktop application. It is meant to be used as a development environment for the components in the packages in this monorepo. It is not intended to be used as a seed/starter application. You should use the [Create React App template](https://www.npmjs.com/package/@itwin/cra-template-desktop-viewer) to seed a new application.
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Configuration
```

Comment:
So i hate to even suggest this, partially because its going to lead to more confusion, but maybe you want to add a link to the desktop starter tutorial or copy a lot of that in here.

Because when you register a new app you also need to setup the auth redirect uris. (I could be wrong about this, and this couldve changed with the auth changes keith made to be done in the backend) 
![image](https://user-images.githubusercontent.com/11051042/123123448-d56c2180-d414-11eb-9196-507c647682a8.png)

Reply:
And if its not needed, we need to update developer.bentley to hide those fields

Reply:
The goals is for the CRA template to replace that tutorial (or be used in it) though. Once we have a CRA template, we will add more clearly defined steps to register your app (similar to the web CRA template README)

Reply:
The requirement for those redirects is still required so we should point to something.

Reply:
this is just the developer app and it uses the default redirect uri:

`  public static defaultRedirectUri = "http://localhost:3000/signin-callback";
  private _configuration: AuthorizationServiceConfiguration | undefined;
  private _tokenResponse: TokenResponse | undefined;
  private _tokenStore?: ElectronTokenStore;
  public get tokenStore() { return this._tokenStore!; }

  public constructor(config?: NativeAppAuthorizationConfiguration) {
    super(config);
  }

  public get redirectUri() { return this.config?.redirectUri ?? ElectronAuthorizationBackend.defaultRedirectUri; }`

I don't think we need to add steps to change that since the intention of this app is strictly for developing features in the package. We can add additional steps to the CRA template.